### ISSUE TITLE
Strict checking input values causes issues

### ISSUE THREAD

**Issue by danielpeintner** [eclipse/thingweb.node-wot#547]
see use-case described in https://github.com/w3c/wot-thing-description/issues/1234.

use-case: Action should work with no value (default kicks in) or with a given value.

No matter what kind of data schema I set one or the other use-case doesn't work and fails with

`[binding-http] HttpServer on port 8080 got internal error on invoke '/countdown/actions/startCountdown': Invalid value according to DataSchema`

see 
https://github.com/eclipse/thingweb.node-wot/blob/43eb73f6032497d45b3af3d7b977448220669c84/packages/examples/src/scripts/countdown.ts#L61-L68

**Comment by relu91** [comment]
yeah, I would have imagined that this wouldn't work 😭 . The solution is to define a new keyword in the TD spec as @egekorkan is suggesting in w3c/wot-thing-description#1234 and disable ajv if no-value is passed AND it is optional.
